tests/test_proxy_connect.py

```diff
@@ -1,63 +1,78 @@
 import json
 import os
+import re
+from subprocess import Popen, PIPE
+import sys
```

Comment:
:lipstick: I think the `subprocess` import should be below `sys`, being in the `from … import …` form. See https://stackoverflow.com/a/20763446/939364